src/graphql/error-map.ts

```diff
@@ -27,7 +27,7 @@ const assertUnreachable = (x: never): never => {
 
 export const mapError = (error: ApplicationError): CustomApolloError => {
   const errorName = error.name as ApplicationErrorKey
-  let message = error.message || errorName || ""
+  let message: string = errorName || ""
```

Comment:
we need to handle properly the error but the message needs to be passed (we are using it in a lot of errors). IMO error message needs to remain here

Reply:
Yea I was trying to figure out where we use these for the unhandled ones (this only counts for unhandled errors in the map). What's an example of where we use the message? Maybe we can figure out something that makes sense without exposing internal details accidentally.